packages/clarity-js/src/layout/dom.ts

```diff
@@ -236,10 +237,10 @@ function privacy(node: Node, value: NodeValue, parent: NodeValue): void {
             metadata.privacy = Privacy.Text;
             break;
         case tag === Constant.TextTag:
-            // If it's a text node belonging to a STYLE or TITLE tag or one of SCRUB_EXCEPTIONS, then capture content
+            // If it's a text node belonging to a STYLE or TITLE tag or one of scrub exceptions, then capture content
             let pTag = parent && parent.data ? parent.data.tag : Constant.Empty;
-            let pSelector = parent && parent.selector ? parent.selector[Selector.Stable] : Constant.Empty;
-            metadata.privacy = pTag === Constant.StyleTag || pTag === Constant.TitleTag || override.some(x => pSelector.indexOf(x) >= 0) ? Privacy.None : current; 
+            let pSelector = parent && parent.selector ? parent.selector[Selector.Beta] : Constant.Empty;
```

Comment:
Minor suggestion: Define a constant one place that is the production one and use that to refer the one in use. That way flipping beteween them should be a one-line change that ripples to everywhere. Too easy otherwise to miss them buried. Then make sure there is no code anywhere referring to Selector.{Alpha/Beta}

Reply:
Good point. Fixed it.